internal/kojiapi/api/openapi.yml

```diff
@@ -115,6 +147,12 @@ components:
         koji_task_id:
           type: integer
           example: 203143
+    ComposeLogs:
+      required:
+        - image_logs
+      properties:
+        image_logs:
+          type: array
```

Comment:
If an array is returned we should document what each part of the array is. I thought you intended to just return one opaque blob? `image_logs` makes it sound like this is one blob per image build, but we also will have koji-init and koji-finalize logs, where would they go?

Reply:
The array elements are undocumented because each of those is an opaque blob – one for each image request that was sent, in the same order. For now, they contain the contents of the job result, but we don't specify that.

The top level has `image_logs` so that we can put other things next to it (for example output from init/finalize jobs) if we want to.

Reply:
As promised, you explained why I was wrong. This makes perfect sense to me then.